app/components/channel_drawer/teams_list/teams_list.js

```diff
@@ -45,14 +45,15 @@ class TeamsList extends PureComponent {
     }
 
     selectTeam = (team) => {
-        const {actions, closeChannelDrawer, currentTeamId} = this.props;
-        if (team.id === currentTeamId) {
-            closeChannelDrawer();
-        } else {
-            actions.handleTeamChange(team);
-
-            closeChannelDrawer();
-        }
+        requestAnimationFrame(() => {
+            const {actions, closeChannelDrawer, currentTeamId} = this.props;
+            if (team.id === currentTeamId) {
+                closeChannelDrawer();
+            } else {
+                actions.handleTeamChange(team);
+                closeChannelDrawer();
```

Comment:
Can this be moved to be after the `if/else` to avoid duplication?

Reply:
Yeah I can improve this, probably left it because of what was there before